src/System.Private.ServiceModel/src/System/IdentityModel/Tokens/X509SecurityToken.cs

```diff
@@ -52,9 +52,15 @@ internal X509SecurityToken(X509Certificate2 certificate, string id, bool clone,
 
             _id = id;
 
+#if TARGETS_WINDOWS 
             _certificate = clone ? new X509Certificate2(certificate.Handle) : certificate;
```

Comment:
We already have an overload on the constructor to specify whether to clone. We don't need to make any changes in this class, just in the consumer.